### PR TITLE
Docs and db migration for #460

### LINKED ISSUE #440: Check for open tasks
I'd like to see a **zappr** check on [open tasks](https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments) in the pull request description. As checking/un-checking a task item is notified as "edit comment" an integration might be possible.

### PR DESCRIPTION
**Pull request by prayerslayer**
Fixes #440 for real, follow-up on #460

### CODE REVIEW

File: config/system.yaml [hunk 1]
```diff
@@ -103,6 +103,7 @@ ZAPPR_WELCOME_TEXT: |
   ## Pull Request guidelines
 
     * Block a PR until it has the required set of labels (e.g. `reviewed`).
+    * Block a PR as long as it has unfinished tasks
```
Comment (sebastianpoeplau) [inline config/system.yaml]: Is it a problem that this new bullet point is not terminated with a period? :P

**Comment by sebastianpoeplau** [comment]
Is it a bug or a feature that zappr doesn't pick up the 👍 in my review above?

File: config/system.yaml (hunk 1 above)
Comment (prayerslayer) [inline config/system.yaml]: meh. yeah. will fix.

**Comment by prayerslayer** [comment]
i think it's a "feature" as it wasn't a regular comment, but a review.

**Comment by sebastianpoeplau** [comment]
👍

**Comment by prayerslayer** [comment]
👍

**Pull request merged**